src/util/RegistrationFormFactory.js

```diff
@@ -125,10 +125,10 @@ define([
     var inputOptions = SchemaFormFactory.createInputOptions(schemaProperty);
     if (inputOptions.type === 'select') {
       inputOptions = _.extend(inputOptions, {
-        label: schemaProperty.get('description')
+        label: schemaProperty.get('title')
```

Comment:
Are we going to use `schemaProperty.get('title')` going forward to match with the API? Will this be a problem if we try to use an older version of the widget?

Reply:
Confirmed with UX that they always expected title and not description to be shown as placeholder/label.